<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register Code Generator v0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_datasheet_dump"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_JSON_configuration"/>
    <addaction name="actionSave_configuration"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGenerate_Code">
    <property name="title">
     <string>Generate Code</string>
    </property>
    <addaction name="actionConfiguration"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGenerate_Code"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_datasheet_dump">
   <property name="text">
    <string>Open datasheet dump...</string>
   </property>
  </action>
  <action name="actionOpen_JSON_configuration">
   <property name="text">
    <string>Open JSON configuration...</string>
   </property>
  </action>
  <action name="actionSave_configuration">
   <property name="text">
    <string>Save configuration...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit...</string>
   </property>
  </action>
  <action name="actionBit_Shifting">
   <property name="text">
    <string>Bit Shifting...</string>
   </property>
  </action>
  <action name="actionBit_Field">
   <property name="text">
    <string>Bit Field...</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
